<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">padding: 2px</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: white; border-color: black; border-width: 3px; border-style: solid; padding: 3px</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>7</number>
      </property>
      <property name="topMargin">
       <number>7</number>
      </property>
      <property name="rightMargin">
       <number>7</number>
      </property>
      <property name="bottomMargin">
       <number>7</number>
      </property>
      <item>
       <widget class="QLabel" name="generateLabel">
        <property name="styleSheet">
         <string notr="true">font: 15px;
font-weight: bold;
border-width: 0px</string>
        </property>
        <property name="text">
         <string>Generate the SQL scripts to edit the notifications of the following ICPs:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="icpLabel">
        <property name="styleSheet">
         <string notr="true">font: 15px;
font-weight: bold;
border-width: 0px</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="destinationLabel">
        <property name="styleSheet">
         <string notr="true">font: 15px;
font-weight: bold;
border-width: 0px</string>
        </property>
        <property name="text">
         <string>Please specify a folder to save the scripts:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="destinationLineEdit">
          <property name="styleSheet">
           <string notr="true">border-width: 1px</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="browseButton">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 12px;
font-weight: bold;
border-style: solid;
border-width: 2px;
border-color: black;
border-radius: 3px;
padding: 8px;
width: 55px</string>
          </property>
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="generateButton">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
border-style: solid; 
border-width: 2px; 
border-radius: 10px; 
border-color: black; 
background-color: rgb(200, 183, 255); 
font: 15px;
font-weight: bold; 
padding: 8px;}
QPushButton:hover{
background-color:rgb(0,0,128);
color: white;
}
</string>
        </property>
        <property name="text">
         <string>Generate</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>220</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
